pyqt-apps/tests/test_configuration_window.py

```diff
@@ -0,0 +1,277 @@
+"""Configuration window tests."""
```

Comment:
wow, `unittest`? @gu1lhermelp, if you keep raising the bar this way it will be difficult for us to follow up!

Reply:
I think it is a good idea that our windows have tests, even if simple, so we can we can be sure there are no broken windows on master